### ISSUE TITLE
Outdated?

### ISSUE THREAD

**Issue by rok9ru** [orangelynx/ic2-extra-fuels#1]
I tryed to install addon and got next:

```
 [main/INFO] [FML]: Forge Mod Loader version 14.23.5.2811 for Minecraft 1.12.2 loading
....
 [Client thread/INFO] [FML]: FML has found a non-mod file ic2-extra-fuels-1.0-all.jar in your mods directory. It will now be injected into your classpath. This could severe stability issues, it should be removed if possible.
```

**Comment by orangelynx** [comment]
Hi and sorry for the late reply. The mod was developed for forge 2808 originally, but I'll see what I can do.

**Comment by orangelynx** [comment]
Fixed with [release v1.1](https://github.com/orangelynx/ic2-extra-fuels/releases/tag/v1.1)
